server-master/src/main/scala/com/aliyun/emr/rss/service/deploy/master/Master.scala

```diff
@@ -421,12 +421,12 @@ private[deploy] class Master(
     context.reply(OneWayMessageResponse)
   }
 
-  private def handleGetClusterLoadStatus(context: RpcCallContext): Unit = {
-    val (_, _, _, result) = getClusterLoad
+  private def handleGetClusterLoadStatus(context: RpcCallContext, numPartitions: Int): Unit = {
+    val (_, _, _, result) = getClusterLoad(numPartitions)
```

Comment:
Why we need so much return values, please remove useless return

Reply:
the [getClusterLoad] is common function, we may need the top three return-values in the other place. I can separate the value through creating new function.